src/components/oculus-touch-controls.js

```diff
@@ -309,14 +309,14 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
   },
 
   onButtonChanged: function (evt) {
+    var button = this.mapping[this.data.hand].buttons[evt.detail.id];
+    if (!button) { return; }
```

Comment:
Not sure I fully understand the bug now. Since we were already checking the button existence in `onButtonChangedV3` and early returing. this and old code should be equivalent

Reply:
This jsfiddle is the simplest demonstration of what I mean https://jsfiddle.net/1xgnzv58/3/

That is what the current code does. If I have a touch v3 controller, it won't fire the `triggerchanged` event.

Reply:
thanks!